app/src/main/java/com/framgia/fsalon/screen/scheduler/SchedulerViewModel.java

```diff
@@ -49,11 +54,14 @@
     private boolean mIsLoadingMore = false;
     private Calendar mCalendar;
     private FragmentManager mFragmentManager;
-    private int mStartDate = OUT_OF_INDEX;
+    private int mStartDate = Utils.createTimeStamp(SchedulerViewModel.TabFilter.TAB_TODAY);
```

Comment:
WARNING: [UnnecessaryFullyQualifiedName] Unnecessary use of fully qualified name 'SchedulerViewModel.TabFilter.TAB_TODAY' due to existing static import 'com.framgia.fsalon.screen.scheduler.SchedulerViewModel.TabFilter.TAB_TODAY'
https://pmd.github.io/pmd-5.5.1/pmd-java/rules/java/imports.html#UnnecessaryFullyQualifiedName